src/main/g8/build.sbt

```diff
@@ -17,6 +17,6 @@ lazy val root = (project in file(".")).
 
       "com.typesafe.akka" %% "akka-http-testkit"        % akkaHttpVersion % Test,
       "com.typesafe.akka" %% "akka-actor-testkit-typed" % akkaVersion     % Test,
-      "org.scalatest"     %% "scalatest"                % "3.0.8"         % Test
+      "org.scalatest"     %% "scalatest"                % "3.2.3"         % Test
```

Comment:
We should definitely update this to 3.1, but I'm not sure about 3.2: Akka still depends on 3.1 (https://github.com/akka/akka/issues/29368) and letting this diverge might cause trouble.

Reply:
Ok, will decrease to 3.1.X, moment

Reply:
Downgraded to 3.1.4